cypress/Shared/CQLLibraryPage.ts

```diff
@@ -4,18 +4,48 @@ export class CQLLibraryPage {
 
     public static readonly createCQLLibraryBtn = '[data-testid="create-new-cql-library-button"]'
     public static readonly cqlLibraryNameTextbox = '[data-testid="cql-library-name-text-field"]'
+    public static readonly cqlLibraryModelDropdown = '[class="MuiSelect-select MuiSelect-outlined MuiOutlinedInput-input MuiInputBase-input MuiInputBase-inputSizeSmall css-182didf"]'
```

Comment:
can you request some IDs for these elements?

Reply:
ok